src/components/widgets-settings/dates-picker/dates-picker.directive.coffee

```diff
@@ -4,12 +4,13 @@ module.directive('settingDatesPicker', ($templateCache, $filter, ImpacWidgetsSvc
   return {
     restrict: 'A',
     scope: {
-      parentWidget: '='
-      deferred: '='
+      parentWidget: '=?'
+      deferred: '=?'
       fromDate: '=from'
       toDate: '=to'
       keepToday: '='
       onUse: '&?'
+      onChangeCb: '&?'
```

Comment:
naming: `onChange` to be consistent with `on-click`, `on-select`, `on-use`, etc.